src/components/tableOfTasks/TaskTable.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Typography, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Box,
+} from '@mui/material';
+import { TaskRow } from '../index.ts';
+import theme from '../../theme/index.ts';
+
+export default function TaskTable() {
+  const data = [
+    {
+      id: 1,
+      title: 'hello',
+      description: 'jndknvlskdksdcscskdn',
+      created_at: '2023-06-19T05:36:48.216Z',
+      due_date: '2023-07-04T22:00:00.000Z',
+      project: 'PROJECT2',
+      priority: 'hight',
+      color: '#F8524B',
+      section: 'To-Do',
+    },
+    {
+      id: 2,
+      title: 'hello',
+      description: 'jndknvlskdksdcscskdn',
+      created_at: '2023-06-19T05:36:48.216Z',
+      due_date: '2023-07-04T22:00:00.000Z',
+      project: 'PROJECT2',
+      priority: 'hight',
+      color: '#F8524B',
+      section: 'To-Do',
+    },
+    {
+      id: 3,
+      title: 'hello',
+      description: 'jndknvlskdksdcscskdn',
+      created_at: '2023-06-19T05:36:48.216Z',
+      due_date: '2023-07-04T22:00:00.000Z',
+      project: 'PROJECT2',
+      priority: 'medium',
+      color: '#FF8800',
+      section: 'To-Do',
+    },
+    {
+      id: 4,
+      title: 'hello',
+      description: 'jndknvlskdksdcscskdn',
+      created_at: '2023-06-19T05:36:48.216Z',
+      due_date: '2023-07-04T22:00:00.000Z',
+      project: 'PROJECT2',
+      priority: 'low',
+      color: '#06C270',
+      section: 'To-Do',
+    },
+  ];
+  return (
+    <TableContainer component={Box} bgcolor="transparent">
+      <Table sx={{ minWidth: 650 }} size="small" aria-label="a dense table">
```

Comment:
what about the spacing ya Khaled?😆

<img width="1289" alt="image" src="https://github.com/GSG-G13/trakio/assets/52599778/c0a839d1-fb48-4ee0-83ec-02a77261ead0">


<img width="1163" alt="image" src="https://github.com/GSG-G13/trakio/assets/52599778/0039b8a6-38a3-4e5d-9dc8-33377a4542c3">